helpers/metadata.ts

```diff
@@ -49,19 +50,24 @@ function compileFormatsMetadata(formats: Format[]): PackFormatMetadata[] {
   });
 }
 
-function compileFormulasMetadata(formulas: Formula[]): PackFormulaMetadata[] {
+export function compileFormulasMetadata(formulas: Formula[]): PackFormulaMetadata[] {
   return formulas.map(compileFormulaMetadata);
 }
 
-function compileFormulaMetadata(formula: TypedPackFormula): PackFormulaMetadata {
-  const {execute, ...rest} = formula;
-  return rest;
+export function compileFormulaMetadata(formula: TypedPackFormula): PackFormulaMetadata {
+  const {execute, validateParameters, ...rest} = formula;
+  const validateParametersAsFormula = wrapMetadataFunction(validateParameters);
```

Comment:
Hmm not sure why wrapMetadataFunction is spread around, for all the other metadata formulas it's done only once in api.ts. So would imagine we would only do the wrapping once, in `makeFormula()` probably. Does that work or is there some complication?

Reply:
Looked at getSchema for example; on SyncTableDef, it is typed as MetadataFormula. It's only MetadataFunctionDef on DynamicOptions which is what gets used in the builder. I guess I need to change what the builder/legacy helpers takes as an argument and what CommonPackFormulaDef has to be more similar. Trying that now